pmd-apex/src/main/resources/category/apex/documentation.xml

```diff
@@ -8,4 +8,23 @@
     <description>
 Rules that are related to code documentation.
     </description>
+
+    <rule name="ApexDoc"
+          since="9.9.9"
```

Comment:
Which version?

Reply:
At the time, the next release would be 6.7.0